src/test/java/org/cloudfoundry/promregator/config/OAuth2XSUAAAuthenticationConfigurationTest.java

```diff
@@ -10,38 +10,23 @@
 
 	@Test
 	public void testTokenServiceURLBackwardCompatibility() {
-
-		/*
```

Comment:
These tests does not make that much sense now. They now ensure that the url we hand over via configuration is not touched.